doc/source/force-constants.rst

```diff
@@ -87,6 +87,20 @@ This is a simple example, but the same applies to more realistic 3D structures.
 This transformation is done automatically when reading from Phonopy.
 If reading force constants from another program, there is a function to help with this transformation, see :ref:`Reading From Other Programs<fc_read_other_programs>`.
 
+**Polar Force Constants**
+
+From equation 78 in `Gonze & Lee, 1997 <https://doi.org/10.1103/PhysRevB.55.10355>`_:
+
+.. math::
+
+  C = C^{SR} + C^{DD}
+
+Where :math:`C` is the 'long-ranged' force constants matrix, :math:`C^{SR}` is the 'short-ranged' force constants matrix, and :math:`C^{DD}` is the long-ranged dipole part (which is non-zero for materials with non-zero Born effective charges and dielectric permittivity tensor).
+Euphonic requires :math:`C^{SR}`, whereas some codes (e.g. Phonopy) output :math:`C` so must be converted.
```

Comment:
Have changed some of the docs in [dbc8790](https://github.com/pace-neutrons/Euphonic/pull/241/commits/dbc8790b711d669032444e9a92b4487ed309ca99), hopefully this is clearer? In this case do we want to rename `from_long_ranged_dipole_fc`?

Reply:
The new text works for me! Would `from_total_fc` be confusing? It seems most consistent with docs, and its own docstring can clarify. Seems like it might become less clear in other contexts :disappointed:

Reply:
Yeah, I agree it's consistent but could be confusing until someone reads the docstring. How about `from_total_fc_with_dipole`? Seems like a mouthful but is shorter than `from_long_ranged_dipole_fc` at least...

Reply:
`from_total_fc_with_dipole` seems ok. `from_fc_including_dipole` might be another option to consider. I'm not sure that it's less ambiguous!

Reply:
Ok, updated in [36a5a95](https://github.com/pace-neutrons/Euphonic/pull/241/commits/36a5a95a6bad5079619f2d27a67671393269aa71)